ckanext/ozwillo_organization_api/plugin.py

```diff
@@ -216,3 +221,118 @@ def get_actions(self):
             'create-ozwillo-organization': create_organization,
             'delete-ozwillo-organization': delete_organization
         }
+
+
+# Used for tests purposes
```

Comment:
can't it be in another file if it's just for test purposes ?

Reply:
Not really as the class implementation has to be done in the plugin.py file.
It makes sure that organizations created by the old way (via ckan) are also treated so I can test it on my machine.
I can remove it in test/prod files if you wish.

Reply:
ok, leave it here